internal/validation/validation_test.go

```diff
@@ -19,8 +19,8 @@ func assertError(t *testing.T, givenError, expectedError error) {
 	}
 }
 
-// TestValidateUserCredentials tests ValidateSignUpRequest function.
-func TestValidateUserCredentials(t *testing.T) {
+// TestValidateSignUpRequest tests ValidateSignUpRequest function.
```

Comment:
No need to write comments to test functions their names are self-descriptive

Reply:
> 
> 
> No need to write comments to test functions their names are self-descriptive

This just a reflex :)